test/units/modules/test_yum.py

```diff
@@ -128,19 +139,38 @@
 glibc.x86_64            2.17-157.el7_3.1  xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"""
 
 
-unwrapped_output_rhel7_obsoletes = unwrapped_output_rhel7 + wrapped_output_rhel7_obsoletes_postfix
```

Comment:
Please avoid reformats like these, as they make backports harder, convolute `git blame`, and make it hard to see the actual change that you're making.

Reply:
reverted